sdk/boms/azure-spring-boot-bom/CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # Release History
 
 ## 3.6.0 (2021-06-23)
-
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.5.0](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.0/spring-boot-dependencies-2.5.0.pom).
+- Upgrade to [Azure Core 1.17.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md#1170-2021-06-07).
+- Upgrade to [Azure Identity 1.3.1](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/CHANGELOG.md#131-2021-06-08).
+- Upgrade to [Azure Key Vault Secret 4.3.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md#430-2021-06-17).
+- Upgrade to [Azure MSAL 1.10.1](https://github.com/AzureAD/microsoft-authentication-library-for-java/releases/tag/v1.10.1).
+- Upgrade to [Azure Service Bus JMS 0.0.7](https://search.maven.org/artifact/com.microsoft.azure/azure-servicebus-jms/0.0.7/jar).
```

Comment:
Same for this one. 

And if you plan to delete this, please check all other entries.